### PR TITLE
TeamCity Badge 

### PR DESCRIPTION
**Pull request by nghe0001**
# Description

Added Team City Badge

Description of the task you completed and/or features added.

## Acceptance Criteria

Get the acceptance criteria for the story and/or task you are working on and put it all here.

## Test Instructions

1. Steps on...
2. how to test...
3. or inspect this...
4. feature manually.

## Help Requested

- Things that you (the submitter)...
- want reviewers to pay very close...
- attention to when they review this PR.

## Checklist

- [ ] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Alpha Site
+
+[![Build Status](https://teamcity.dts-stn.com/app/rest/builds/buildType:(id:Dev_AlphaSite_Build)/statusIcon)](https://teamcity.dts-stn.com/viewType.html?buildTypeId=Dev_AlphaSite_Build&guest=1)
+
+
```
Comment (Moro-Code) [inline README.md:5]: can we also add a badge for the link to the application

**Review by Moro-Code — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (nghe0001) [reply README.md:5]: k i will update that.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# Alpha Site
+
+[![Build Status](https://teamcity.dts-stn.com/app/rest/builds/buildType:(id:Dev_AlphaSite_Build)/statusIcon)](https://teamcity.dts-stn.com/viewType.html?buildTypeId=Dev_AlphaSite_Build&guest=1)
+[![link](https://img.shields.io/badge/link-%F0%9F%94%97-brightgreen)](https://alphasite.main.dev.dts-stn.com/)
```
Comment (pcraig3) [inline README.md:4]: Pretty sure the link here is `https://alphasite.dev.dts-stn.com/`, not `https://alphasite.main.dev.dts-stn.com/`

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+# Alpha Site
+
+[![Build Status](https://teamcity.dts-stn.com/app/rest/builds/buildType:(id:Dev_AlphaSite_Build)/statusIcon)](https://teamcity.dts-stn.com/viewType.html?buildTypeId=Dev_AlphaSite_Build&guest=1)
```
Comment (pcraig3) [inline README.md:3]: Do I need an account to TeamCity?

**Review by pcraig3 — COMMENTED** [review]
Just 1 quick change but it looks good after that.

**Review by pcraig3 — APPROVED** [review]

**Review by pcraig3 — APPROVED** [review]

**Review by pcraig3 — APPROVED** [review]

**Pull request merged**